packages/client3/components/design-system/toggle/toggle.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+import { Toggle } from "./toggle";
+
+export default {
+  title: "FAD/Components/Toggle",
+  component: Toggle,
+} as ComponentMeta<typeof Toggle>;
+
+export const ToggleStory: ComponentStory<typeof Toggle> = (args) => (
+  <Toggle {...args} />
+);
+
+let value = true;
+ToggleStory.args = {
+  onChange: () => {
+    value = !value;
+  },
+  states: {
+    selectedState: value,
+    state1: "Backer",
+    state2: "Artist",
```

Comment:
in my opinion, this is not quite clear what state is active, I mean

`state1, state2, selectedState: true,` what is selected?

myabe change the `selectedState` to `isState1Selected`